Vagrantfile

```diff
@@ -90,8 +90,11 @@ Vagrant.configure(2) do |config|
       # Install app dependencies in virtual environment as vagrant user
       sudo -H -u vagrant sh -c '. ~/venv/bin/activate && pip install -U pip && pip install wheel'
       sudo -H -u vagrant sh -c '. ~/venv/bin/activate && cd /vagrant && pip install -r requirements.txt'
-    SHELL
 
+      # Create .env file if it doesn't exist
+      sudo -H -u vagrant sh -c 'cd /vagrant && if [ ! -f .env ]; then cp dot-env-example .env; fi'
```

Comment:
It seems like `dot-env-example` doesn't exist yet, so this line is throwing an error. Would you mind looking into this? Thank you!

Reply:
Thanks :) 
Do you think I should revert the changes to the Vagrant file or do you think I should add a dot-env-example file (perhaps to maintain consistency between the team members' .env files)? Let me know your thoughts!

Reply:
I am not entirely sure, I think there's no harm in creating a `dot-env-example` since there's no sensitive information in it. 
But then we may as well just include the `.env`, but it may appear slightly amateurish to not gitignore a `.env` file. 
Personally, I'd be fine with either approach : )

Reply:
That's how I see it too: a tradeoff between simplicity and best practice. I noticed that missing information in the .env file leads to a failed bdd test case. For now, I will adhere to the lab so that we don't run into problems across the team. If anyone thinks differently please let me know.